task09-reactjs/src/components/cards/CardItem.js

```diff
@@ -0,0 +1,24 @@
+import classes from './CardItem.module.css'
+
+function CardItem(props) {

```

Comment:
Use destructuring to get the props directly:
`function CardItem({name, image, description, ...etc})`